MapboxGLAndroidSDKTestApp/build.gradle

```diff
@@ -1,5 +1,5 @@
 apply plugin: 'com.android.application'
-apply from: "${rootDir}/gradle/native-build.gradle"
+apply from: "../gradle/native-build.gradle"
```

Comment:
could be `{projectDir}/../`? Even though shouldn't be any differece?

Reply:
that works as well